### DISCUSSION TITLE
Why react-native-tab-view is faster than react navigation tabs by a large margin?

### DISCUSSION THREAD

**Discussion by arasrezaei** [react-navigation/react-navigation#11368] [Q&A] [answered]
Hi there,
I've tested this to libraries on galaxy note 4.
React native tab view never lag or delay but react native tabs is slow.
What's the problem hers?

**Comment by okwasniewski** [comment, marked as answer]
Hey, 

The issue is that `react-native-tab-view` can do a lot more memoization of TabBar components when changing screens. The current implementation of tab-view doesn't allow us to apply those optimizations to `material-top-tabs`. However, I'm currently working on API changes for tab-view that will allow us to achieve the same level of performance so stay tuned for an RFC with new API.
